pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApiOffloadTest.java

```diff
@@ -71,8 +71,6 @@ private void testOffload(String topicName, String mlName) throws Exception {
         LedgerOffloader offloader = mock(LedgerOffloader.class);
         when(offloader.getOffloadDriverName()).thenReturn("mock");
 
-        doReturn(offloader).when(pulsar).getManagedLedgerOffloader();
```

Comment:
@jiazhai I removed it because of the new lazy load feature of offload policies even for non defined offloaders. WDYT about removing this?